src/CoWSwapEthFlow.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8;
+
+import "./libraries/EthFlowOrder.sol";
+import "./interfaces/ICoWSwapEthFlow.sol";
+import "./mixins/CoWSwapOnchainOrders.sol";
+
+/// @title CoW Swap ETH Flow
+/// @author CoW Swap Developers
+contract CoWSwapEthFlow is CoWSwapOnchainOrders, ICoWSwapEthFlow {
+    using EthFlowOrder for EthFlowOrder.Data;
+
+    /// @dev An order that is owned by this address is an order that has not yet been assigned.
+    address public constant NO_OWNER = address(0);
```

Comment:
Yes! I made it public because I wanted to make it accessible if other people use this contract, but it doesn't make sense that this field is accessible from the contract. I moved it to the order library.